### PR TITLE
Fix json-appender to handle logs with parameters

### PR DESCRIPTION
**Pull request by manoskouvarakis**

### CODE REVIEW

File: src/initializers/log4js/json-appender.ts [hunk 1]
```diff
@@ -48,10 +48,13 @@ const getContextObject = data => {
...
-export function configure() {
-  return jsonAppender();
+function configure(config, layouts) {
+  let layout = layouts.messagePassThroughLayout;
+  return jsonAppender(layout);
 }
+
+module.exports.configure = configure;
+
+// export function configure() {
```
Comment (nikosd23) [inline src/initializers/log4js/json-appender.ts]: Why is this commented out?

File: src/initializers/log4js/json-appender.ts [hunk 2]
```diff
@@ -23,11 +23,11 @@ export const createErrorLog = logEvent => {
   };
 };
 
-export const createValidLog = logEvent => {
+export const createValidLog = (layout, logEvent) => {
```
Comment (nikosd23) [inline src/initializers/log4js/json-appender.ts]: What if you don't pass layout?

File: src/initializers/log4js/json-appender.ts (hunk 1 above)
Comment (manoskouvarakis) [reply src/initializers/log4js/json-appender.ts]: Fixed

File: src/initializers/log4js/json-appender.ts (hunk 2 above)
Comment (manoskouvarakis) [reply src/initializers/log4js/json-appender.ts]: This method is exported only for testing purposes. So layout will be passed from json appender initialisation

File: test/env.ts [hunk 3]
```diff
@@ -2,3 +2,4 @@ process.env.LOG_LEVEL = 'fatal';
 process.env.RIVIERE = 'false';
```
Comment (manoskouvarakis) [inline test/env.ts]: This line should be removed with the new enabled property

Comment (nikostoulas) [reply test/env.ts]: Done

**Pull request merged**